src/overlay/test/OverlayTestUtils.cpp

```diff
@@ -0,0 +1,55 @@
+// Copyright 2022 Stellar Development Foundation and contributors. Licensed
+// under the Apache License, Version 2.0. See the COPYING file at the root
+// of this distribution or at http://www.apache.org/licenses/LICENSE-2.0
+
+#include "overlay/test/OverlayTestUtils.h"
+#include "medida/metrics_registry.h"
+#include "overlay/OverlayManager.h"
+#include "overlay/OverlayMetrics.h"
+
+namespace stellar
+{
+
```

Comment:
should these functions be in their own namespace?

Reply:
Done!